packages/gestalt/src/boxTransforms.js

```diff
@@ -470,7 +470,7 @@ export function buildStyles<T: Object>({
   // eslint-disable-next-line no-restricted-syntax
   for (const prop in props) {
     if (
-      // $FlowFixMe[method-unbinding]
+      // $FlowExpectedError[method-unbinding]
```

Comment:
There isn't really a way around this. Flow doesn't like binding Object's `hasOwnProperty` to a different `this`. ESLint doesn't like calling `Object.hasOwnProperty` directly, since it might not exist or may have been overwritten.